pkg/image/resolve_export_test.go

```diff
@@ -0,0 +1,10 @@
+package image
+
+// export private functions for testing only
+
+var FetchImagesFromCharts = fetchImagesFromCharts
```

Comment:
Hmm is this needed? If your test file is on the same package with actual file, I think you can use private functions too.

Reply:
The problem I ran into is reslove_test.go depends on kdm creating a cyclic import.  If the tests exist in image_test you cannot import private functions.